src/services/findUserById.ts

```diff
@@ -0,0 +1,6 @@
+import usersFromServer from '../api/users';
+import { User } from '../types/user';
+
+export const findUserById = (id: number): User => (
+  usersFromServer.find(user => user.id === id) as User
```

Comment:
find can return the undefined
```suggestion
  usersFromServer.find(user => user.id === id) || null
```